docs/en/api/FS-API.md

```diff
@@ -296,8 +288,8 @@ this policy preference exists only for local workers, not remote workers; remote
 blocks to the highest tier.
 
 By default, data is written to the top tier. Users can modify the default setting through the
-`alluxio.user.file.write.tier.default` [configuration]({{ '/en/operation/Configuration.html' | relativize_url }})
-property or override it through an option to the 
+`alluxio.user.file.write.tier.default` [property]({{ '/en/reference/Properties-List.html' | relativize_url }})
```

Comment:
i learned yesterday when reviewing beinan's PR that you can anchor tag every single property in the properties list page -> https://docs.alluxio.io/os/user/stable/en/reference/Properties-List.html#alluxio.user.file.write.tier.default
```suggestion
`alluxio.user.file.write.tier.default` [property]({{ '/en/reference/Properties-List.html' | relativize_url }}#alluxio.user.file.write.tier.default)
```